doc/content/reference/email-templates/available.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Available Templates"
+description: ""
+weight: 3
+---
+
+The following email templates are currently used by The Things Stack:
+
+Full Name | Identifier | Scope | Additional fields
+----------|------------|-------|------------------
+API Key changed | api_key_changed | Sent when the rights of an API Key have been changed. | `Identifiers` and `Rights`
+API Key created | api_key_created | Send when an API Key has been created. | `Identifiers` and `Rights`
+Collaborator changed | collaborator_changed | Sent when the rights of a collaborator have been changed. | `Collaborator`
+Password changed | password_changed | Sent when the the password of an user has been changed.
+Temporary password | temporary_password | Sent when an temporary password has been requested for an user. | `TemporaryPassword`
+Account validation | validate | Sent when a user is created in order to validate their email address. | `ID` and `Token`
+
+The following fields can be used inside all of the email templates:
```

Comment:
They're not always all available, i.e. `User` is only set if we're sending an email to a User.

Reply:
Aren't `User.Name` and `User.Email` always available since they represent the recipient of the email ?

Reply:
We also send emails to `ContactInfo` objects that don't have a `User`